packages/react-native-reanimated/Common/cpp/reanimated/NativeModules/NativeReanimatedModule.cpp

```diff
@@ -24,6 +24,7 @@
 #include "CollectionUtils.h"
 #include "EventHandlerRegistry.h"
 #include "FeaturesConfig.h"
+#include "REAWorkletRuntimeDecorator.h"
```

Comment:
We're using `REA` prefix only in Obj-C codebase; otherwise it should always be `Reanimated`, i.e. `ReanimatedCommitHook`, `ReanimatedHermesRuntime` etc.
```suggestion
#include "ReanimatedWorkletRuntimeDecorator.h"
```